gap/attr.gi

```diff
@@ -3358,6 +3358,157 @@ function(D)
+  # size of the least set S that contain u and v
+  # such that any (u, v)-path (that is, a path with source u and target v)
+  # passes through S. If two vertices are adjacent, then the local connectivity
+  # is infinity by convention. The minimum cut with source u and target v is
+  # the least number of edges that need to be removed so that there is no
+  # longer a (u, v)-path.
+
+  # Let u and v be non-adjacent vertices in `digraph`. Because of the
+  # correspondence of paths in `digraph` and `doubled_digraph`, any such set S
+  # for `digraph` corresponds to a set of edges E_S (obtained by replacing the
+  # vertex w by the edge (2*w-1, 2*w)) in `doubled_digraph` whose removal
+  # disconnects 2*u from 2*v-1.
+  # Conversely, it can be shown that the smallest set of edges disconnecting
+  # 2*u from 2*v-1 has the same cardinality as E_S. This is because, whenever
+  # we remove any edge (2*s, 2*t-1) from `doubled_digraph` with the goal of
+  # disconnecting 2*u from 2*v-1, it is always just as
+  # efficient or more efficient to remove the edge (2*s-1, 2*s) instead, since
+  # any path from 2*u to 2*v-1 utilizing (2*s, 2*t-1) in `doubled_digraph` must
+  # pass through (2*s-1, 2*s) by construction. Note that u and v are
+  # non-adjacent by assumption, so it cannot be the case that
+  # (2*s, 2*t-1) = (2*u, 2*v-1).
+  # It follows that, for non-adjacent vertices, the local connectivity of u and
+  # v in `digraph` is equal to the minimum cut with source 2*u-1 and target 2*v
+  # in `doubled_digraph`.
+
+  # By the max-flow min-cut theorem, the size of the minimum cut with source u
+  # and target v equals the maximum flow between u and v see Wikipedia below:
+  # https://en.wikipedia.org/wiki/Max-flow_min-cut_theorem
+  # Hence we can compute local vertex connectivity by repeated calls to the
+  # max_flow function below:
+  max_flow := {D, source, target} ->
+    Sum(DigraphMaximumFlow(D, source, target)[source]);
+
+  # The vertex connectivity is the minimum of the local vertex connectivity
+  # over all pairs of vertices. However, it can be computed a bit more
+  # cleverly by utilizing some theory to reduce the number of pairs considered.
+  # In this function we implement Algorithm 11 from Abdol-Hossein
+  # Esfahanian's ``Connectivity Algorithms'' which can be found at
+  # https://www.cse.msu.edu/~cse835/Papers/Graph_connectivity_revised.pdf
+  # In particular, we reduce the number of local vertex connectivity
+  # computations to n-d-1 + d*(d-1)/2 where n is the total number of vertices
+  # and d is the minimum degree of any vertex.
+  v := PositionMinimum(OutDegrees(digraph));
+  neighbours_v := OutNeighboursOfVertex(digraph, v);
+  # TODO: is adjacency function the best way to check adjacency?
+  adjacency_function := DigraphAdjacencyFunction(digraph);
+  kappa_min := fail;
+  for u in DigraphVertices(digraph) do
+    if u <> v and not adjacency_function(v, u) then
```

Comment:
Does the `u <> v` check here mean that you can avoid removing all loops above? I haven't checked.

Also, I think the most efficient thing you can do in general is simply:
```suggestion
    if u <> v and not u in neighbours_v then
```